code/modules/xgm/xgm_gas_mixture_presets.dm

```diff
@@ -0,0 +1,18 @@
+
+/datum/gas_mixture/vacuum/New()
+	..()
+	// does nothing, type just for clarity/organization
```

Comment:
This does something, it's another proc call that doesn't have any reason to be there, you can just define it instead of overriding this proc forcing calls to traverse this before reaching the parent

Reply:
Probably fair, but still not ideal when you can just

`/datum/gas_mixture/vacuum`

To define it, without anything else; it's also a good practice not to overload procs if you're not doing anything with it usually (can be done for debugging purposes and probably I myself forgot to take some away afterwards, but that's more of a failure on my part to remember to remove them than something you want to do)

Reply:
![image](https://github.com/user-attachments/assets/20a73ad6-1201-408e-ade4-06525d47decf)
oh yeah I'll just do this